### PR TITLE
Set bounds manually

### PR DESCRIPTION
**Pull request by brotkiste**
See #11

### CODE REVIEW

**Comment by allartk** [comment]
I had little time. 
Maybe a getter and setter method to set the bounds dynamic would make it more complete. Let me know. Otherwise i will merge!

**Comment by brotkiste** [comment]
Yes, I'll look into that. Do you already have any setters for options where I can orient on?
I've also been thinking about combining savewhat you see and manually set bounds, so that one button saves the set bounds and the second what you see

**Comment by allartk** [comment]
Hi, eg setLayer

**Comment by brotkiste** [comment]
Like this?

File: index.html [hunk 1]
```diff
@@ -42,7 +42,8 @@
         }).addTo(map);
         //add buttons to save tiles in area viewed
         var control = L.control.savetiles(baseLayer, {
-            'zoomlevels': [13, 16], //optional zoomlevels to save, default current zoomlevel
+            'zoomlevels': [10], //optional zoomlevels to save, default current zoomlevel
+            bounds:L.latLngBounds([[47.67186,11.78489],[47.78617,12.44476]]),
```
Comment (allartk) [inline index.html]: can you alter this line? Otherwise the example on http://allartk.github.io/leaflet.offline/ will always download this.

**Comment by allartk** [comment]
I will merge after editing index.html

**Pull request merged**